components/shared/richText/RichTextElement.tsx

```diff
@@ -116,11 +117,12 @@ export const createDefaultResolvers = (
         case contentTypes.action.codename:
           return <CTAButton reference={componentItem as Action} />;
         case contentTypes.fact.codename:
-          return (
-            <FactComponent
-              item={componentItem as Fact}
-              isReversed={false}
-            />
+          return (
+            // incrementing componentIndex ensures zigzag pattern of facts
+            <FactComponent
+              item={componentItem as Fact}
+              isReversed={componentIndex++ % 2 !== 0}

```

Comment:
This is rather complicated. What about using css variables together with the `odd:` or `even:` classes instead? https://www.thomasledoux.be/blog/tailwind-setting-using-css-variables-react

Reply:
https://github.com/kontent-ai/sample-app-next-js/issues/60